### DISCUSSION TITLE
Error using gatsby-cli 3.2

### DISCUSSION THREAD

**Discussion by crydestaahl** [gatsbyjs/gatsby#30702] [Help]
Hi! 

I'm having some big troubles when trying to use the gatsby-cli version 3.2.0. 

![Skärmavbild 2021-04-05 kl  12 35 10](https://user-images.githubusercontent.com/39494432/113565592-7ebc2c00-960b-11eb-8961-a876fe7febc9.png)

I'm currently using:
macOS Catalina 10.15.7
Node 15.11.0
npm 7.6.3

I'm a regular user of GatsbyJS but since the upgrade to 3.0.0 I'm unable to use the client or even run my projects locally. 

If someone could help me with this it would mean a lot!! :)

**Comment by LekoArts** [comment]
Please give more details about your problem.

**Reply by crydestaahl** [reply]
I think this could be some kind of Node problem. When I'm trying to run one of my projects locally I get an error saying node-sass is incompatible. I tried npm rebuild node-sass and then get a long list of errors (I've upgraded node since I posted the issue). 

![Skärmavbild 2021-04-06 kl  15 22 38](https://user-images.githubusercontent.com/39494432/113717584-0d07df00-96ec-11eb-871b-df78b240d2c6.png)
![Skärmavbild 2021-04-06 kl  15 22 54](https://user-images.githubusercontent.com/39494432/113717602-109b6600-96ec-11eb-890e-4d85a611b39d.png)

**Reply by crydestaahl** [reply]
Also, in the Gatsby client I tried the --verbose flag, but it didn't give any info...
